docs/fips.html

```diff
@@ -27,6 +27,18 @@
 	    (revision 2.0.16).
 	    </p>
 
+            <p>
+            Neither validation will work with any release other than 1.0.2.
+            The OpenSSL project is no longer maintaining either the 1747
+            or the 2398 module. This includes adding platforms to those
+            validations.
+            We are starting work on a new validation, based on the upcoming
+            1.1.1 release.
+            That module will have a fixed set of validations.
```

Comment:
"a small set of validated operational environments (platforms)" 

is better than "fixed set of validations"